tests/test_examples.py

```diff
@@ -241,7 +241,7 @@ def to_test(model_name: str, multi_card: bool, deepspeed: bool, example_name: st
             return True
         elif "bridgetower" in model_name and IS_GAUDI2:
             return True
-        elif "falcon" in model_name and IS_GAUDI2 and not fsdp:
+        elif "falcon" in model_name and IS_GAUDI2 and not fsdp and not fp8:
             return True
         elif "bloom" in model_name and deepspeed and not IS_GAUDI2:
```

Comment:
We should modify line 211 above as follows:
```
if (fsdp or fp8) and not IS_GAUDI2:
```
to make sure the test will not be run on Gaudi1.

Reply:
Done